jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/FrameHandler.java

```diff
@@ -391,11 +395,22 @@ public Duration getIdleTimeout()
                 return Duration.ZERO;
             }
 
+            @Override
+            public Duration getWriteIdleTimeout()
+            {
+                return Duration.ZERO;
+            }
+
             @Override
             public void setIdleTimeout(Duration timeout)
             {
             }
 
+            @Override
+            public void setWriteIdleTimeout(Duration timeout)
+            {
+            }
```

Comment:
perhaps throw unsupportedOperationException of the value is non zero?

Reply:
I don't that should be done here in `CoreSession.Empty`, validation is not done for other methods in this class which take parameters